### PR TITLE
Data hub accessiblity fixes

### PR DESCRIPTION
**Pull request by sagar-lcm**
## Description
The following ticket's changes are present in the PR:

- https://massgov.atlassian.net/browse/DP-26730
- https://massgov.atlassian.net/browse/DP-26734
- https://massgov.atlassian.net/browse/DP-26747
- https://massgov.atlassian.net/browse/DP-26746
- https://massgov.atlassian.net/browse/DP-26749

### CODE REVIEW

**Review by ygannett — APPROVED** [review]
LGTM.  Thank you for all a11y improvements.

**Comment by ygannett** [comment]
@sag13684 Can you look into https://output.circle-artifacts.com/output/job/3444f156-a874-4fd3-9c63-f1aa0a5b39e0/artifacts/0/backstop/data/html_report/index.html with https://app.circleci.com/pipelines/github/massgov/mayflower/4591/workflows/9cd5c780-473a-43cd-8fc0-15e6a155e019/jobs/51586?
They seem to be related.  Once this is fixed, the PR can be merged.  Thank you!

**Comment by ygannett** [comment]
@sag13684 Can you look into the 2 failures?  They seem to be related.  As the CSS gets fixed, the backstop error should go away.  Once it's fixed, the PR can be merged.  Thank you!

https://output.circle-artifacts.com/output/job/3444f156-a874-4fd3-9c63-f1aa0a5b39e0/artifacts/0/backstop/data/html_report/index.html

File: changelogs/DP-26730.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#  - project: Enter one or multiple (comma separated) valid project prefix(es) - see ___PROJECT___ e.g. React, Patternlab
+#    component: Enter one or multiple (comma separated) valid component name(s) - see ___COMPONENT___ e.g. Color
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+#___EXAMPLE___
+Added:
+  - project: React
+    description: Add autocomplete attribute to HeaderSearch molecule.
```
Comment (clairesunstudio) [inline changelogs/DP-26730.yml]:
```suggestion
    description: Add autocomplete attribute to HeaderSearch molecule. (#1715)
```
can you please add the PR number to the changelogs, like this?

**Comment by clairesunstudio** [comment]
Something seems to broken the two-column styles of the LinkList component in backstop --> https://output.circle-artifacts.com/output/job/3f2c9368-0646-49f8-8ca2-a53172e4e807/artifacts/0/backstop/data/html_report/index.html

**Comment by clairesunstudio** [comment]
You can ignore the backstop test failure on LinkList, it's introduced by this PR https://github.com/massgov/mayflower/pull/1705

File: changelogs/DP-26734.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+#  - project: Enter one or multiple (comma separated) valid project prefix(es) - see ___PROJECT___ e.g. React, Patternlab
+#    component: Enter one or multiple (comma separated) valid component name(s) - see ___COMPONENT___ e.g. Color
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+#___EXAMPLE___
+Added:
+  - project: React
+    description: Added visually hidden form title to SearchBannerForm.
```
Comment (clairesunstudio) [inline changelogs/DP-26734.yml]:
```suggestion
    description: Added visually hidden form title to SearchBannerForm. (#1715)
```

File: changelogs/DP-26743.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+#  - project: Enter one or multiple (comma separated) valid project prefix(es) - see ___PROJECT___ e.g. React, Patternlab
+#    component: Enter one or multiple (comma separated) valid component name(s) - see ___COMPONENT___ e.g. Color
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+#___EXAMPLE___
+Added:
+  - project: React
+    description: Added role attribute to InputCheckbox.
```
Comment (clairesunstudio) [inline changelogs/DP-26743.yml]:
```suggestion
    description: Added role attribute to InputCheckbox. (#1715)
```

File: changelogs/DP-26746.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+#  - project: Enter one or multiple (comma separated) valid project prefix(es) - see ___PROJECT___ e.g. React, Patternlab
+#    component: Enter one or multiple (comma separated) valid component name(s) - see ___COMPONENT___ e.g. Color
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+#___EXAMPLE___
+Added:
+  - project: React
+    description: Added clear all button context for Tags molecule.
```
Comment (clairesunstudio) [inline changelogs/DP-26746.yml]:
```suggestion
    description: Added clear all button context for Tags molecule. (#1715)
```

File: changelogs/DP-26747.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+#  - project: Enter one or multiple (comma separated) valid project prefix(es) - see ___PROJECT___ e.g. React, Patternlab
+#    component: Enter one or multiple (comma separated) valid component name(s) - see ___COMPONENT___ e.g. Color
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+#___EXAMPLE___
+Added:
+  - project: React
+    description: Added disabled property for SelectBox.
```
Comment (clairesunstudio) [inline changelogs/DP-26747.yml]:
```suggestion
    description: Added disabled property for SelectBox. (#1715)
```

File: changelogs/DP-26749.yml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+#  - project: Enter one or multiple (comma separated) valid project prefix(es) - see ___PROJECT___ e.g. React, Patternlab
+#    component: Enter one or multiple (comma separated) valid component name(s) - see ___COMPONENT___ e.g. Color
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+#___EXAMPLE___
+Added:
+  - project: React
+    description: Added back to top property for Pagination molecule.
```
Comment (clairesunstudio) [inline changelogs/DP-26749.yml]:
```suggestion
    description: Added back to top property for Pagination molecule. (#1715)
```

File: changelogs/DP-26752.yml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+#  - project: Enter one or multiple (comma separated) valid project prefix(es) - see ___PROJECT___ e.g. React, Patternlab
+#    component: Enter one or multiple (comma separated) valid component name(s) - see ___COMPONENT___ e.g. Color
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+#___EXAMPLE___
+Added:
+  - project: React
+    description: Allows extra properties to be passed to BreadcrumbItem.
```
Comment (clairesunstudio) [inline changelogs/DP-26752.yml]:
```suggestion
    description: Allows extra properties to be passed to BreadcrumbItem. (#1715)
```

File: changelogs/DP-26730.yml (hunk 1 above)
Comment (sagar-lcm) [reply changelogs/DP-26730.yml]: Done.

**Review by clairesunstudio — APPROVED** [review]

File: changelogs/DP-26734.yml (hunk 2 above)
Comment (sagar-lcm) [reply changelogs/DP-26734.yml]: Done.

**Comment by clairesunstudio** [comment]
The test failure is not related to this PR. I created a separated ticket to address that on develop

**Comment by clairesunstudio** [comment]
patternlab backstop test failure is a false positive

**Pull request merged**